lib/intercom/request.rb

```diff
@@ -151,7 +151,11 @@ def raise_application_errors_on_failure(error_list_details, http_code)
       }
       case error_code
       when 'unauthorized', 'forbidden', 'token_not_found'
-        raise Intercom::AuthenticationError.new(error_details['message'], error_context)
+        if error_details['message'].include? 'suspended application'
```

Comment:
Do we return `suspended application` in the response for deleted apps too?

Reply:
Yes we do, but I think it might be safer to use the error type instead of the message string!